reactor-core/src/main/java/reactor/core/scheduler/ElasticScheduler.java

```diff
@@ -107,7 +104,15 @@ public ScheduledExecutorService createUndecoratedService() {
 
 	@Override
 	public void start() {
-		throw new UnsupportedOperationException("Restarting not supported yet");
+		if (!shutdown) {
```

Comment:
What are the concurrency guarantees of `start()`? Perhaps we should do a CAS here?

Reply:
explicitly documented as non-thread-safe, so this should be allright